adapters/syslog/syslog.go

```diff
@@ -76,7 +77,14 @@ type SyslogAdapter struct {
 
 func (a *SyslogAdapter) Stream(logstream chan *router.Message) {
 	for message := range logstream {
-		err := a.tmpl.Execute(a.conn, &SyslogMessage{message, a})
```

Comment:
From local testing using rsyslog, it seems that the golang templates package doesn't do any internal buffering, and writes data to the socket as soon as it is ready. This seems to result in multiple udp packets being transmitted, which rsyslog interprets as separate (and malformed) log messages.